.circleci/config.yml

```diff
@@ -265,6 +265,7 @@ jobs:
           name: Generate checksum of data that populates the test database
           command: |
               $TEST_HOME/local/runtime-config/db_content_fingerprint.sh > /tmp/db_data_md5key
```

Comment:
can also use `tee` here so you can see output:
```
 $TEST_HOME/local/runtime-config/db_content_fingerprint.sh | tee /tmp/db_data_md5key
```
https://shapeshed.com/unix-tee/

Reply:
@inodb That is a good idea, but out of scope here. Let's do this in an other PR.